documentation/components/Skeleton.mdx

```diff
@@ -0,0 +1,83 @@
+---
+name: Skeleton
+menu: Components
+route: /components-skeleton
+---
+
+import { Playground, Props } from 'docz'
+import Component from 'react-component-component'
+import { Skeleton } from '../../src'
+import { Row } from './helpers'
+
+# Skeleton
+
+```javascript
+import Skeleton from 'mfcl/Skeleton'
+```
+
+## Description
+
+The Skeleton component allows you to a build out placeholder Skeleton layouts.
+
+## Properties
+
+<Props of={Skeleton} />
+
+## Basic usage
+
+<Playground>
+  <Skeleton width="25%" />
```

Comment:
just for examples sake could you showcase the various ways this prop could be passed?
like as a number and then as a string with different units?

Reply:
Just now added. How does this look? 

![image](https://user-images.githubusercontent.com/11052295/106179954-4eff2c80-6161-11eb-978f-9cd6203949dc.png)